configs/nextflow.base.config

```diff
@@ -62,50 +62,65 @@ profiles {
 	}
 }
 
+singularity {
+	runOptions = '--bind /' // Bind directories to image for jasen run to access i.e. directories containing sample reads 
+	cacheDir = "${params.containerDir}/"
+	enabled = true
+
+	env.NXF_APPTAINER_LOCALCACHEDIR="${params.workDir}"
```

Comment:
Will APPTAINER - variables work for Singularity CE? Otherwise, perhaps good to keep both the singularity and apptainer versions for some time?